src/js/wrapper.ts

```diff
@@ -98,11 +96,34 @@ export const NATIVE = {
       },
     };
 
+    if (getPlatform() === CordovaPlatformType.Android) {
```

Comment:
This is something we'll need to address in both RN and Cordova

Reply:
Tried to get rid of this in #208, see process description.